Sources/MessagingPush/PushHandling/iOSPushEventListener.swift

```diff
@@ -37,7 +27,7 @@ protocol PushEventListener: AutoMockable {
  - During the native iOS SDK's initialization, the SDK's digraph instance is re-created. All objects (and singletons) in that old digraph instance are deleted from memory.
```

Comment:
I think we won't have this issue with `DIGraphShared` that is being added for CDP work.